src/firefly/plans/xafs_scan.py

```diff
@@ -214,11 +214,12 @@ def update_total_time(self):
         self.time_calculation_signal.emit()
 
 
-class XafsScanDisplay(display.FireflyDisplay):
+class XafsScanDisplay(regions_display.PlanDisplay, display.FireflyDisplay):
```

Comment:
Don't need to inherit from FireflyDisplay since PlanDisplay already is a subclass of FireflyDisplay.